content/chainguard/libraries/python/build-configuration.md

```diff
@@ -163,6 +172,24 @@ Refer to the official documentation for [configuring authentication with
 pip](https://pip.pypa.io/en/stable/topics/authentication/) if you are not using
 [.netrc for authentication](/chainguard/libraries/access/#netrc).
 
+When using [direct access](#direct-access) to the Chainguard Libraries for
+Python with pip in the configuration files you must replace any `/` in the
+username value `CG_PULLTOKEN_USERNAME` with `_` and ensure the `simple` context
+is used for the URL. The password value `CG_PULLTOKEN_PASSWORD` remains
+unchanged.
+
+Example for `requirements.txt`:
+
+```
```

Comment:
These all render as `Output` boxes. You could label it `requirements.txt` though that will probably capitalize it to `Requirements.txt`. You could also label it `example`